src/NumericalAlgorithms/RootFinding/TOMS748.hpp

```diff
@@ -17,6 +17,15 @@
 
 namespace RootFinder {
 namespace toms748_detail {
+// Original implementation of TOMS748 is from Boost:
+//  (C) Copyright John Maddock 2006.
+//  Use, modification and distribution are subject to the
+//  Boost Software License, Version 1.0. (See accompanying file
+//  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
You're missing the actual license file.

Reply:
I added the file. Much to my surprise it looks like installed versions of Boost also don't necessarily include it, but I agree we should include the license for it.